rosen_backend/videoUpload/admin.py

```diff
@@ -1,7 +1,8 @@
 from django.contrib import admin
 
 # Register your models here.
-from videoUpload.models import VideoModel,ImageModel
+from videoUpload.models import VideoModel,ReferenceImageModel,ImageFrameModel
 
 admin.site.register(VideoModel)
```

Comment:
Question to Ali:
What does the admin do here? Is it for the localhost:8000/admin page?

Reply:
Yes, this registers the models on the admin page so that you can see the models there